diff --git a/audio/out/ao_wasapi_utils.c b/audio/out/ao_wasapi_utils.c
index a696040dba..9bcdba3dc3 100644
--- a/audio/out/ao_wasapi_utils.c
+++ b/audio/out/ao_wasapi_utils.c
@@ -22,6 +22,15 @@
 #include <windows.h>
 #include <errors.h>
 #include <ksguid.h>
+
+#ifndef WAVE_FORMAT_DTS
+#define WAVE_FORMAT_DTS 0x0008
+#endif
+
+#ifndef WAVE_FORMAT_DOLBY_AC3_SPDIF
+#define WAVE_FORMAT_DOLBY_AC3_SPDIF 0x0092
+#endif
+
 #include <ksmedia.h>
 #include <avrt.h>
 
